src/main/java/net/openhft/chronicle/core/io/TracingReferenceCounted.java

```diff
@@ -55,6 +55,16 @@ public StackTrace createdHere() {
         return createdHere;
     }
 
+    @Override
+    public void addReferenceChangeListener(ReferenceChangeListener referenceChangeListener) {
+        // This is taken care of by VanillaReferenceCounted
+    }
+
+    @Override
+    public void removeReferenceChangeListener(ReferenceChangeListener referenceChangeListener) {
+        // This is taken care of by VanillaReferenceCounted
+    }
```

Comment:
This is how I got around `DualReferenceCounted`. I'm not happy with it but also not sure what the best solution is here.